variables.tf

```diff
@@ -178,6 +161,23 @@ variable "tags" {
   default     = {}
 }
 
+variable "vpc_id" {
+  description = "The VPC that is used for the instances."
+  type        = "string"
+}
+
+variable "availability_zone_proxy" {
```

Comment:
This variable has no any references in code

Reply:
fixed